lib/ae_mdw/db/contract.ex

```diff
@@ -844,9 +844,9 @@ defmodule AeMdw.Db.Contract do
     create_txi = Origin.tx_index!(state, {:contract, contract_pk})
 
     m_ct_from =
-      Model.aexn_contract_from_transfer(index: {create_txi, from_pk, txi, i, to_pk, value})
+      Model.aexn_contract_from_transfer(index: {create_txi, from_pk, txi, to_pk, i, value})
 
-    m_ct_to = Model.aexn_contract_to_transfer(index: {create_txi, to_pk, txi, i, from_pk, value})
+    m_ct_to = Model.aexn_contract_to_transfer(index: {create_txi, to_pk, txi, from_pk, i, value})
```

Comment:
We want to sort transfers by {txi, idx}, not by {txi, pubkey} where pubkey is not really relevant for sorting. The only thing we need here is a migration to reindex the AexnContractToTransfer and FromTransfer, which wasn't reindexed here https://github.com/aeternity/ae_mdw/commit/b32a95ca36f385e5277f04ef5edfe83e1c434d22#diff-845330bd4377f6efe9113d9a750cacdc9c00c7706c4134b59d6d12fe92e6a9c6